Hearthstone Deck Tracker/Utility/ValueMoments/ValueMomentConstants.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hearthstone_Deck_Tracker.Utility.ValueMoments
+{
+	public class ValueMomentsConstants
```

Comment:
By moving to abstract properties, you should be able to get rid of all of these. If something is used between classes, like Battlegrounds vs Mercenaries actions, you can just create an abstract class that declares this and the other actions can inherit from this.

Otherwise ask me/Alex again.